### ISSUE TITLE
Reproducibility Issue

### ISSUE THREAD

**Issue by rsk2327** [mdiephuis/SimCLR#10]
I am unable to reproduce the results for CIFAR that you have achieved. The final validation accuracy is around 30% while train accuracy is at 60%.

Im also curious about the loss value during the training of the feature model. Even after 100 epochs, the loss value remains approx. stagnant at 0.0155.  Is this expected?

**Comment by mdiephuis** [comment]
I'll do a run from scratch when I can access my gpus again net week and report back.

**Comment by mdiephuis** [comment]
@rsk2327 You are right. We are looking into it. Thanks!

**Comment by rsk2327** [comment]
Has this been resolved?

**Comment by mdiephuis** [comment]
It has been resolved with the removal of LARS, the results in the readme have been updated.

**Comment by sourabhsugandhi** [comment]
Hi @mdiephuis ,

Is the above problem fixed, because i am facing similar problem when i tried with same training procedure that is mentioned in README

**Issue 01:**
![image](https://user-images.githubusercontent.com/31852515/122748004-57d7d280-d2a9-11eb-972d-803a68dcdd75.png)
**Script used  - train_classifier.py**
I am able to achieve Batch accuracy of ~31% for Training and ~37% for validation and training stopped at 199th epoch

**Issue 02:**
![image](https://user-images.githubusercontent.com/31852515/122748998-725e7b80-d2aa-11eb-9d0b-02f604b392ef.png)

**Script used  - train_features.py**
I am able to achieve Loss of ~23% for Training and ~22% for validation and training stopped at 149th epoch

Can you please help me with solution or is this the expected issue. even i am unable to reproduce the results for CIFAR that you have achieved so far
